server/datastore/mysql/software.go

```diff
@@ -1609,6 +1639,7 @@ func (ds *Datastore) ListSoftwareForVulnDetection(
 			goqu.I("s.version"),
 			goqu.I("s.release"),
 			goqu.I("s.arch"),
+			// TODO: validate that we don't need the new fields here (browser, extension_id, etc.)
```

Comment:
@getvictor This one already selected only a subset of `software` columns, just want to make sure we do NOT require `extension_id` to be added?

Reply:
Correct. We don't consider specific chrome-based browsers right now, although we could in the future.